source/devicedefender/ReportTaskWrapper.h

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef AWS_IOT_DEVICE_CLIENT_REPORTTASKWRAPPER_H
+#define AWS_IOT_DEVICE_CLIENT_REPORTTASKWRAPPER_H
+
+#include <aws/iotdevicedefender/DeviceDefender.h>
+
+using namespace std;
+using namespace Aws;
+
+namespace Aws
+{
+    namespace Iot
+    {
+        namespace DeviceClient
+        {
+            namespace DeviceDefender
+            {
+
```

Comment:
Comments please.

Reply:
As in for the doxygen? They're there I think at bottom?

Reply:
I don't see any comments in source/devicedefender/ReportTaskWrapper.h 
There should be some description at least.